src/formulations/ComplexMaxwell/complex_maxwell_formulation.hpp

```diff
@@ -59,10 +59,11 @@ class ComplexMaxwellFormulation : public hephaestus::FrequencyDomainEMFormulatio
   const std::string _loss_coef_name;
 };
 
-class ComplexMaxwellOperator : public ProblemOperator
+class ComplexMaxwellOperator : public EquationSystemProblemOperator
```

Comment:
Does this need to be an `EquationSystemProblemOperator`, or could it just remain a ProblemOperator? The actual operator builds its own complex weak form here with a ParSesquilinearForm